hack/scale_out_poc/two_TP/nginx.conf

```diff
@@ -50,70 +50,59 @@ http {
-                set $tenant_api_one {{ arktos_api_protocol }}://{{ tenant_partition_one_ip }}:{{ arktos_api_port }};
-                set $tenant_api_two {{ arktos_api_protocol }}://{{ tenant_partition_two_ip }}:{{ arktos_api_port }};
-
-                #################################################################
-                # forward nodes related requests to resource partition
-                #################################################################
-
-                location ~ ^/api/v1/nodes?(.*) {
-                    proxy_pass $resource_api;
-                }
-
-                location ~ ^/apis/coordination.k8s.io/v1/leases?(.*) {
-                    proxy_pass $resource_api;
-                }
-
-                location ~ ^/apis/([^/])*/([^/])*/tenants/system/namespaces/kube-node-lease/leases?(.*) {
-                    proxy_pass $resource_api;
-                }
-
-                #################################################################
-                # forward the request of system-tenants or no tenants specificed back to its source partition
-                #################################################################
-
-                location ~* ^/apis/([^/])*/([^/])*/(!tenants)?(.*) {
-                    proxy_pass http://$remote_addr:8080;
-                }
-
-                location ~* ^/api/([^/])*/(!tenants)?(.*) {
-                    proxy_pass http://$remote_addr:8080;
-                }
-
-                location ~ ^/apis/[a-z0-9_.-]+/[a-z0-9_.-]+/tenants/system?(.*) {
-                    proxy_pass http://$remote_addr:8080;
-                }
-
-                location ~ ^/api/[a-z0-9_.-]+/tenants/system?(.*) {
-                    proxy_pass http://$remote_addr:8080;
-                }
-
-                #################################################################
-                # forward the tenant-specific requests to its own tenant-partition
-                #################################################################
-
-                location ~* ^/apis/[a-zA-Z0-9_.-]+/[a-zA-Z0-9_.-]+/tenants/[a-m0-9_.-]+/(.*) {
-                    proxy_pass $tenant_api_one;
-                }
-                location ~* ^/apis/[a-zA-Z0-9_.-]+/[a-zA-Z0-9_.-]+/tenants/[n-z_.-]+/(.*) {
-                    proxy_pass $tenant_api_two;
-                }
-
-                #################################################################
-                # should not hit here. if here, send the request back
-                #################################################################
-                location / {
-                    proxy_pass http://$remote_addr:8080;
-                }
-        } 
+            listen      8888;
+            server_name ip-172-31-26-146 172.31.26.146;
+
+            access_log /var/log/nginx/arktos_access.log;
+            error_log /var/log/nginx/arktos_error.log;
+
+            set $resource_api http://172.31.23.48:8080;
```

Comment:
is there any missing file in this PR, i.e. to replace those hard-coded IP/Ports ?